alf/algorithms/predictive_representation_learner.py

```diff
@@ -230,6 +238,9 @@ def __init__(self,
                 dynamics net. Otherwise, a linear projection will be used to
                 convert the current latent represenation to the initial state for
                 the dynamics net.
+            postprocessor (None|Callable): If provided, will be called as
+                ``postprocessor(latent)`` to get the actual representation,
+                where ``latent`` is the output from encoding_net.
```

Comment:
Is it useful to provide an option to specify postprocessor optimizer, to be structurally similar to ``encoding_net`` and ``dynamics net``? Might not be necessary in most of the cases when view postprocessor as part of the downstream networks, but might be useful in some cases.